easybuild/easyconfigs/h/hanythingondemand/hanythingondemand-3.0.0-intel-2015b-Python-2.7.10.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'PythonPackage'
+
+name = 'hanythingondemand'
+version = '3.0.0'
+
+homepage = 'https://github.com/hpcugent/hanythingondemand'
+description = """HanythingOnDemand (HOD) is a system for provisioning virtual Hadoop clusters over a large physical cluster.
+It uses the Torque resource manager to do node allocation."""
+
+toolchain = {'name': 'intel', 'version': '2015b'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+python = 'Python'
+pythonver = '2.7.10'
+pythonshortver = '.'.join(pythonver.split('.')[:2])
+versionsuffix = '-%s-%s' % (python, pythonver)
+
+# a Python version with netaddr/netifaces/mpi4py + pbs_python and vsc-base/vsc-mympirun is required at runtime
+dependencies = [
+    (python, pythonver),  # provides netaddr, netifaces, mpi4py
+    ('pbs_python', '4.6.0', versionsuffix),
+    ('vsc-mympirun', '3.4.2', versionsuffix + '-vsc-base-2.4.2'),
+    ('setuptools', '1.4.2', '', True),
+]
+
+use_easy_install = True
```

Comment:
shouldn't, but I'm not 100% sure; at least this makes very sure we're using `easy_install`, which is important for `hod dists`, see also https://github.com/hpcugent/hanythingondemand/issues/78